src/downshift.js

```diff
@@ -543,6 +558,8 @@ class Downshift extends Component {
       'aria-label': isOpen ? 'close menu' : 'open menu',
       'aria-expanded': isOpen,
       'aria-haspopup': true,
+      // TODO: is this supposed to be here? This already has `aria-label`
+      'aria-labelledby': this.labelId,
```

Comment:
In particular, I'm wondering about this. cc @cycomachead

Reply:
Hmm, I can't recall exactly why that was there. Looking that the ARAI 1.1 examples (see ex 3), it seems like the existing aria-label should be sufficient: https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/combobox/aria1.1pattern/listbox-combo.html